src/index.ts

```diff
@@ -155,7 +155,7 @@ export function declareAtom<State>(
 
   throwIf(
     initialState === undefined,
-    'Atom "' + atomId + '". Initial state can\'t be undefined',
+    `Atom "${atomId}". Initial state can't be undefined`,
```

Comment:
don't. It's a very slow operation.

Reply:
its exception....

Reply:
It does not matter :) it is Better to do work with strings in the same style.

Reply:
but code will run anyway so better implement string formatting in `throwIf`  if we want better perfomace

Reply:
But for exceptions... Maybe we should talk about it.

Reply:
About better perfomance. See https://gist.github.com/c564436f39a8a1557b5969f405ccb492

Reply:
rather it is http://jsbench.github.io/#c564436f39a8a1557b5969f405ccb492